drivers/volume/portworx/portworx.go

```diff
@@ -217,6 +217,8 @@ func (d *portworx) Init(sched string, nodeDriver string, token string, storagePr
 }
 
 func (d *portworx) RefreshDriverEndpoints() error {
+	// Force update px endpoints
```

Comment:
What this  options is stands for? Why we need this change?

Reply:
@stgleb RefreshDriverEndpoints() function we are using it here to update list of storage driver endpoints since we want to switch to a different PX cluster. If this variable is not set to true, further function calls skip updating list of storage nodes.
For e.g., If we trace function calls further, we are calling [getPXnodes()](https://github.com/portworx/torpedo/blob/d95ca6922716b1886015d10ed5af3d7c5a4db9fb/drivers/volume/portworx/schedops/k8s-schedops.go#L728) which in turn calls getNodeManager() which will return existing nodeManager (manager to current cluster) instead of returning a new nodeManager for cluster we want to switch to.